jmxtrans-output/jmxtrans-output-influxdb/src/main/java/com/googlecode/jmxtrans/model/output/InfluxDbWriterFactory.java

```diff
@@ -95,12 +93,21 @@ public InfluxDbWriterFactory(
 		this.retentionPolicy = StringUtils.isNotBlank(retentionPolicy) ? retentionPolicy : DEFAULT_RETENTION_POLICY;
 
 		this.resultAttributesToWriteAsTags = initResultAttributesToWriteAsTags(resultTags);
-
+		this.tags = initCustomTagsMap(tags);
```

Comment:
This could be replaced by `ImmutableSet.copyOf(firstNotNull(tags, Collections.emptyMap()))`